### PR TITLE
Cleanup for Deprecated ActivityType

### PR DESCRIPTION
**Pull request by dave2wave**
most solves: #442

### CODE REVIEW

**Comment by dave2wave** [comment]
`ActivityType` is renamed as `LegacyActivityType` and is only used by `StandardActivityType`.

File: nb-engine/nb-engine-core/src/main/java/io/nosqlbench/engine/api/activityapi/core/LegacyActivityType.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2022-2023 nosqlbench
+ * Copyright (c) nosqlbench
```
Comment (msmygit) [inline nb-engine/nb-engine-core/src/main/java/io/nosqlbench/engine/api/activityapi/core/LegacyActivityType.java:2]: let's leave this copyright year intact

**Review by msmygit — COMMENTED** [review]
LGTM. I'd leave this one for @jshook to review as it touches the very core.

File: nb-engine/nb-engine-core/src/main/java/io/nosqlbench/engine/api/activityapi/core/LegacyActivityType.java (hunk 1 above)
Comment (dave2wave) [reply nb-engine/nb-engine-core/src/main/java/io/nosqlbench/engine/api/activityapi/core/LegacyActivityType.java:2]: We've been removing. It is actually irrelevant. You can't reset the year. The year is the original year of the overall product. Copyright protections are time limited and don't reset.

**Review by jshook — COMMENTED** [review]
I'll have a way to completely collapse this down to eliminate the extraneous types. I can tackle it next week. Given that, I'm not sure that the doc changes or naming changes are significant.

**Comment by dave2wave** [comment]
I will leave this open and you can either work from this, or follow your path to fix #442

**Review by jshook — APPROVED** [review]
As a disambiguation, I'm ok with this

**Pull request merged**